docs/cluster-provisioning.md

```diff
@@ -63,6 +63,9 @@ status:
     type: ClusterTemplateValidated
 ```
 
+> [!NOTE]
```

Comment:
You could also mark the hwTemplate (line 34 above) as optional and say "see note below" or something like that.